Backend/Models/Messages/CommandMessage.cs

```diff
@@ -127,5 +134,35 @@ private void ProcessMute()
             SendAccept();
             CLogger.Event("User Muted: " + _target);
         }
+        private void ProcessRemember()
+        {
+            int param1, param2;
```

Comment:
is there a way to avoid this generic naming?

Reply:
done